app/services/umrdr/deposits_okay_service.rb

```diff
@@ -0,0 +1,59 @@
+module Umrdr
+
+  require 'rubygems'
+  require 'net/ldap'
+
+  ENV['LDAPTLS_CACERT'] = '../../../config/incommon_server_ca_cert.crt';
+
+  class DepositsOkayService
+    attr_accessor :ldap, :operation_result, :op_code, :deposits_okay, :uname
```

Comment:
Are all of these attributes required? How many of these attributes need to be used in more than one function?